src/features/settings/Settings.tsx

```diff
@@ -398,13 +398,15 @@ const Settings = () => {
       },
     ]
     if (l1Network === 'solana') {
+      const localNet = __DEV__ ? [{ label: 'Localnet', value: 'localnet' }] : []
       devData = [
         ...devData,
         {
           title: t('settings.sections.dev.solanaNetwork.title'),
           value: solanaNetwork,
           select: {
             items: [
+              ...localNet,
```

Comment:
whoa. What is going on here?

Reply:
lol yeah don't need to spread. Will fix. I am just detecting if we are on a simulator and give us the option to connect to localnet

Reply:
Okay refactored and I am not just pushing to an array if dev mode is on